src/data/set6/champions.ts

```diff
@@ -53,6 +53,22 @@ export default {
 		},
 	},
 
+	[ChampionKey.Kassadin]: {
+		cast: (elapsedMS, spell, champion) => {
+			const target = champion.target
+			const manaReave = champion.getSpellVariable(SpellKey.ManaReave) ?? 0
+			const castTime = champion.getCurrentSpell()?.castTime ?? 0
```

Comment:
The duration of the `DamageReduction` is from the `Duration` variable (`champion.getSpellVariable('Duration')`). We don't need the `castTime` for our part here. (The `Projectile` this creates automatically inherits the `castTime` and missile properties from the spell data.)